.clj-kondo/imports/clj-kondo/slingshot/clj_kondo/slingshot/try_plus.clj

```diff
@@ -0,0 +1,44 @@
+(ns clj-kondo.slingshot.try-plus
```

Comment:
This file (and several other files underneath `.clj-kondo/imports`) are copied from dependencies automatically.